src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -255,11 +255,19 @@ public abstract partial class JsonNode
         public static implicit operator System.Text.Json.JsonNode (uint value) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public static implicit operator System.Text.Json.JsonNode (ulong value) { throw null; }
-        public static System.Text.Json.JsonNode Parse(string json, System.Text.Json.JsonDocumentOptions options = default(System.Text.Json.JsonDocumentOptions), System.Text.Json.DuplicatePropertyNameHandling duplicatePropertyNameHandling = System.Text.Json.DuplicatePropertyNameHandling.Replace) { throw null; }
+        public static System.Text.Json.JsonNode Parse(string json, System.Text.Json.JsonNodeOptions options = default(System.Text.Json.JsonNodeOptions)) { throw null; }
         public string ToJsonString() { throw null; }
         public static bool TryGetNode(System.Text.Json.JsonElement jsonElement, out System.Text.Json.JsonNode jsonNode) { throw null; }
         public void WriteTo(System.Text.Json.Utf8JsonWriter writer) { }
     }
+    public partial struct JsonNodeOptions
```

Comment:
Please document all the public APIs with triple slash comments so we can port them quickly and easily.

Reply:
It's documented, but documentation just isn't shown in the ref file.

Reply:
Ah sorry yes, it's the ref file. My bad :)